File: 58-length-of-last-word/length-of-last-word.cpp
class Solution {
public:
    int lengthOfLastWord(string s) {
        int res = 0;
        bool flag = true;
        for (int i = s.length() - 1; i >= 0; --i) {
            if (flag && s[i] == ' ') {
                continue;
            }
            if (s[i] != ' ') {
                flag = false;
                ++res;
            } else {
                break;
            }
        }
        return res;
    }
};

File: 58-length-of-last-word/length-of-last-word.py
class Solution:
    def lengthOfLastWord(self, s: str) -> int:
        l = 0
        flag = False
        for c in s[::-1]:
            if c == " ":
                if flag:
                    return l
                continue
            else:
                l += 1
                flag = True
        return l